jdr/WFLY-13539-galleon_layer_jdr_support.adoc

```diff
@@ -0,0 +1,131 @@
+= Galleon layer for JBoss Diagnostic Reporting support
+:author:            Yeray Borges
+:email:             yborgess@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The JBoss Diagnostic Reporting (JDR) subsystem enables the gathering of diagnostic data to aid in troubleshooting. JBoss EAP subscribers can provide this information to Red Hat when requesting support.
+
+The diagnosis information is generated by using the following operation:
+
+[source,xml]
+----
+[standalone@localhost:9990 /] /subsystem=jdr:generate-jdr-report()
+----
+
+Users are also able to generate the diagnostic information via command line executing the `jdr.sh` script located at `$WFLY_HOME/bin/` directory. This script accepts the following options:
+
+[source,bash]
+----
+$WFLY_HOME/bin/jdr.sh --help
+usage: jdr.{sh,bat,ps1} [options]
+
+JBoss Diagnostic Reporter (JDR) is a subsystem built to collect information to aid in troubleshooting. The jdr script is a utility for generating JDR reports.
+-c,--config <arg>     Configuration file of the server if it is not running.
+-h,--help             Display this message and exit
+-H,--host <arg>       hostname that the management api is bound to. (default: localhost)
+-p,--port <arg>       port that the management api is bound to. (default: 9990)
+-s,--protocol <arg>   Protocol that is used to connect. Can be remote, http or https (default: http)
+----
+
+The availability of a management interface is optional. If there is not a management interface available, the `jdr.{sh,bat,ps1}` script is still able to extract some information invoking internally the embedded server.
```

Comment:
I'd suggest to s/`internally the embedded server`/`the embedded server internally`/
WDYT?

Reply:
Replaced, thanks